.env.example

```
@@ -0,0 +1,24 @@
+APP_NAME=simplificado
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://localhost
+
+LOG_CHANNEL=stack
+
+DB_CONNECTION=postgres
+DB_HOST=127.0.0.1
+DB_PORT=5432
+DB_DATABASE=simplificado
+DB_USERNAME=simplificado
+DB_PASSWORD=simplificadoPassword
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
+
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
```

Adicionar quebra de linha

```suggestion
REDIS_PORT=6379

```